problems/removeVowels.js

```diff
@@ -6,8 +6,13 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
+function removeVowels(str) {
+let vowels = "aeiouAEIOU"
+let output = str.filter((el) => {
+    return el !== vowels
```

Comment:
Assuming you though el represented a char we don't want to check that a single char does not equal to a whole string of vowels. Instead we want to make sure that vowels does not includes char.